jcapiv1/docs/SystemusersApi.md

```diff
@@ -179,7 +169,7 @@ opts = {
 
 begin
   #List all system users
-  result = api_instance.systemusers_list(content_type, accept, opts)
+  result = api_instance.systemusers_list(opts)
```

Comment:
Double checking we meant to remove `content_type` and `accept` from system users list

Reply:
@adducci those aren't required headers on this endpoint so it's fine to leave them out for now. But, all other endpoints have them in documentation as headers but they aren't required in V1. Let me add them back and we can correct this in a later release for consistency sake. It's confusing as those headers are documented but not required in V1.

Reply:
It looks like all the traits were dumped from this resource in stoplight, so let me look into it.....

Reply:
I've updated Stoplight Docs and checked the YAML In SI, I can push that at the next YAML update for V1. to add back accept and content type headers. since they aren't required this is more a doc inconsistency problem.